### DISCUSSION TITLE
Site not working on Github io

### DISCUSSION THREAD

**Discussion by jwanOo** [community/community#81881] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have tried to deploy my js site on github io. But it seems not to work. 

Here is my repo: https://github.com/jwanOo/daily-verse

this is the page site: https://jwanoo.github.io/daily-verse/

is it because of the index.html? 

Best regards

**Comment by liamcharger** [comment]
I just tested your site, it seems to be working fine for me.

**Reply by jwanOo** [reply]
How does it work for you? You see the content?

**Reply by jane-does-coding** [reply]
It's probably because the icons is not supported on the specific browsers or devices

**Reply by liamcharger** [reply]
Your site was hosting from this file (it's not now):

https://github.com/jwanOo/daily-verse/blob/main/public/index.html

**Comment by jane-does-coding** [comment]
I found the issue! The site isn't working for me and the consosle says that it "Failed to load resource" which appears to be the icon you used in your index.html on line 5: 

    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />

I hope that helps!  :)
Please follow, and star my work, that would help a lot!

**Reply by jwanOo** [reply]
I have deleted the linke but it still doesn't work. I have now added another branch and deployed it from there. But I still see no content. 

This is very strange

**Reply by jane-does-coding** [reply]
Just downloaded the files, yeah it doesn't seem to work without a link either, I don't know why yet but I found a bug in /src/App1.jsx  on the line 77, you wrote className twice:
   
 <cite className="d-block text-center" id="author" className="author">
        <h2>[{snum}:{num}]</h2>
 </cite>

That's not the problem tho, it still doesn't work even after a fix, I'm still looking at it right now

**Reply by jwanOo** [reply]
I beieve it is only the way how I deploy it. I have created a second branch called gh-pages now and have added some files to it. But even this way, it does not work. 

Thank you for your support!

**Comment by dzenebieri** [comment]
add deploy.yml file

**Reply by chien110234** [reply]
Ok
